app/lib/package/backend.dart

```diff
@@ -739,6 +739,7 @@ class PackageBackend {
         filename,
         maxContentLength: maxAssetContentLength,
         maxArchiveSize: UploadSignerService.maxUploadSize,
+        created: DateTime.now().toUtc(),
```

Comment:
didn't we migrate everything to `clock.now()` ?

hmm, funny a few lines up I see  `Stopwatch().` so I'm guessing not.

Reply:
fixed (also indicated via a test :) )